x/permission/types/types.go

```diff
@@ -167,26 +183,13 @@ func (s *Statement) ValidateBasic(resType resource.ResourceType) error {
 	case resource.RESOURCE_TYPE_UNSPECIFIED:
 		return ErrInvalidStatement.Wrap("Please specify the ResourceType explicitly. Not allowed set RESOURCE_TYPE_UNSPECIFIED")
 	case resource.RESOURCE_TYPE_BUCKET:
-		containsCreateObject := false
```

Comment:
maybe moving all `ValidaeBasic` function to `ValidateRuntime` would be better? 
It's kind of complex and unreadable if you split it into two place..

Reply:
`ValidaeBasic` is still used by cli. If we remove it, there will be no validation for cli.